package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@dellstorage/clarity-react",
-    "version": "1.1.20",
+    "version": "1.1.21",
```

Comment:
Good time to move 1.2.0 ?

Reply:
@maheshkulkarni4 @akate1 started with version 1.2.0 now